frontend/src/pages/organization/SecretScanningPage/route.tsx

```diff
@@ -6,7 +6,7 @@ import { SecretScanningPage } from "./SecretScanningPage";
 
 const SecretScanningQueryParams = z.object({
   state: z.string().catch(""),
-  installation_id: z.string().catch("")
+  installation_id: z.coerce.number().optional()
```

Comment:
why do we coerce this into a number? 

for the other, we coerce it as a string
<img width="1152" alt="image" src="https://github.com/user-attachments/assets/49e80403-c251-4f25-afc3-612c9c139227" />